base-notebook/start.sh

```diff
@@ -27,14 +27,23 @@ if [ $(id -u) == 0 ] ; then
     fi
 
     # Enable sudo if requested
-    if [ ! -z "$GRANT_SUDO" ]; then
+    if [ "$GRANT_SUDO" == "1" ]; then
```

Comment:
Good catch, I'll make sure it accepts 'yes'. I mainly wanted to catch 'GRANT_SUDO=0' from accidentally enabling it